src/main/java/dog/pawbook/model/managedentity/owner/Owner.java

```diff
@@ -32,6 +34,8 @@ public Owner(Name name, Phone phone, Email email, Address address, Set<Tag> tags
         this.phone = phone;
         this.email = email;
         this.address = address;
+        this.dogIdSet = new HashSet<>();
+
```

Comment:
There's an extra line break here. Other than that, another naming thing again, this applies to other areas as well, is it better if we use either `id` or `ID` in the naming and don't mix the casing? It helps to make things consistent as well.